spec/system/home_page_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "rails_helper"
```

Comment:
Would it be useful to have something like this to test the claims and placements index pages?

Reply:
Do you still think there is value in this? ☝🏻

Reply:
I'll add in a case to check the page content.

Reply:
Added a temp case while we don't have the login page setup.